packages/tests/server/___testHelpers.ts

```diff
@@ -117,9 +119,21 @@ export function routerToServerAndClientNew<TRouter extends AnyRouter>(
   });
   const trpcClientOptions = {
     links: [
-      httpBatchLink({
-        url: httpUrl,
-        transformer: opts?.transformer as any,
+      splitLink({
```

Comment:
we should also be able to use this with `unstable_httpBatchStreamLink` right? as the responses there aren't batched, just the requests?

Reply:
nvm im dumb it's 1 response just the json chunks being streamed down...